src/common/stores/store-names.ts

```diff
@@ -16,16 +16,8 @@ export enum StoreNames {
     PathSnippetStore,
     UnifiedScanResultStore,
     CardSelectionStore,
-    DeviceStore,
```

Comment:
All of the removed stores are Android-specific. The build will break if any of these were referenced in the existing code.